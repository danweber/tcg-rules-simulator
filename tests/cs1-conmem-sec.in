VERSION:2
# Gain 9 memory. Then, draw 1 card.
SECURITY1:CS1-Confusing
FIELD2:Attack1,DW1-Van Attack2,DW1-Van
GAME:P2 T3 MAIN 3
HAND1:CS1-Confusing CS1-Confusing
DECK1:CS1-02 CS1-02
DUMP:HAND,SECURITY

ECHO: -- P2 attacks, loses turn from memory bomb. (P1 still has 1 card to not deck out.)
CMD:ATTACK Attack1 Player
DUMP:HAND,SECURITY

EXIT
ECHO: -- Try to use again. Can't, not enough memory.
CMD:PLAY CS1-Confusing
DUMP:HAND



