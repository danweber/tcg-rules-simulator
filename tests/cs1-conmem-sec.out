FIELD1:
FIELD2:Attack1,DW1-Vanillamon Attack2,DW1-Vanillamon
SECURITY1:CS1-14
SECURITY2:
HAND1:CS1-14 CS1-14
HAND2:
GAME:P2 T3 MAIN 3
ECHO: -- P2 attacks, loses turn from memory bomb. (P1 still has 1 card to not deck out.)
FIELD1:
FIELD2:Attack1,DW1-Vanillamon,REST Attack2,DW1-Vanillamon
SECURITY1:
SECURITY2:
HAND1:CS1-02 CS1-02 CS1-14 CS1-14
HAND2:
GAME:P1 T4 HATCHING 6
